bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v2/form/FormContainerImpl.java

```diff
@@ -104,6 +105,10 @@ public class FormContainerImpl extends AbstractContainerImpl implements FormCont
     @Nullable
     private String prefillService;
 
+    @ValueMapValue(injectionStrategy = InjectionStrategy.OPTIONAL)
+    @Nullable
+    private String roleAttribute;
```

Comment:
All new properties should have `fd:` prefix in the JCR node, can you change this

Reply:
Added fd: prefix for role attribute.

Reply:
even JCR property should have fd: prefix in it

Reply:
Done